pkg/harness/controller/controller_types.go

```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/pluralsh/deployment-operator/internal/helpers"
 	console "github.com/pluralsh/deployment-operator/pkg/client"
 	"github.com/pluralsh/deployment-operator/pkg/harness/sink"
-	"github.com/pluralsh/deployment-operator/pkg/harness/stackrun"
+	v12 "github.com/pluralsh/deployment-operator/pkg/harness/stackrun/v1"
```

Comment:
maybe `stackrunv1` instead of `v12`?

Reply:
oh, that must have been automatically updated by my IDE during refactoring